server/@types/forms/index.d.ts

```diff
@@ -106,7 +106,12 @@ declare module 'inductionForms' {
   }
 
   export interface WorkInterestTypesForm {
-    workInterestTypes: Array<WorkInterestsValue>
+    workInterestTypes: Array<WorkInterestTypeValue>
     workInterestTypesOther?: string
   }
+
+  export interface WorkInterestRolesForm {
+    workInterestRoles: Map<WorkInterestTypeValue, string>
```

Comment:
This is the map I mentioned in my [previous PR](https://github.com/ministryofjustice/hmpps-education-and-work-plan-ui/pull/468#discussion_r1514323469).